lxqt-base/lxqt-config/lxqt-config-1.2.0-r1.ebuild

```diff
@@ -49,6 +49,14 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
+src_prepare() {
+	if has_version ">=kde-plasma/libkscreen-5.26.90" ; then
```

Comment:
We don't like to do this as it's brittle. If it's purely about fixing an implicit include, we should be able to do this unconditionally?

Reply:
I agree, it is a bit brittle, however I'm not so sure the changes would work with an old kscreen. I will test and if it does, I'll make it unconditional.